### ISSUE TITLE
Toggle blocking of tor nodes

### ISSUE THREAD

**Issue by 0x17de** [AndrewBelt/hack.chat#112]
Just some simple command that will toggle blocking all connections from IPs that were specified per line in some ASCII file.

**Comment by raf924** [comment]
Do it in your own fork

**Comment by AndrewBelt** [comment]
Yeah, it sounds like these are good ideas, but I think they would be better in a new fork, so I'd encourage you to give that a try. And right now, I wouldn't even have enough time to review PRs.

**Comment by 0x17de** [comment]
No need to fork, we already have some other place that works well. But as a moderator it would be nice to at least have some control to handle such situations - at the moment the ban system feels like it could even be removed completely as it'd not make any difference.

**Comment by raf924** [comment]
Although I'd look silly saying "banana time" if I couldn't ban right after

**Comment by 0x17de** [comment]
@AndrewBelt just check out that mass spam happening a lot recently https://www.0x17.de/hackchat/issue2.png  https://www.0x17.de/hackchat/issue3.png

**Comment by raf924** [comment]
I guess now the point is moot. No chat => no spammers
